R/classicProcess.R

```diff
@@ -1541,8 +1544,12 @@ ClassicProcess <- function(jaspResults, dataset = NULL, options) {
   }
 
   .procBootstrapSamplesFootnote(pathCoefTable, procResults, options)
-
-  pathCoefs <- pathCoefs[pathCoefs$op == "~",]
+  
+  # select paths from parameter estimates
+  operators <- if(options[["processModels"]][[modelIdx]][["intercepts"]]) c("~1","~") else "~"
+  pathCoefs <- pathCoefs[pathCoefs$op %in% operators,]
+  pathCoefs[which(pathCoefs$op=="~1"),"rhs"]<-"(Intercept)"
+  pathCoefs <- dplyr::arrange(pathCoefs,desc(op))
```

Comment:
I think it would be better (and consistent with the residual covariance output) to only show estimated intercepts (i.e., those for endogeneous variables). The ordering can also be done without dplyr, then we have a dependecy less.

```suggestion
  operators <- if(options[["processModels"]][[modelIdx]][["intercepts"]]) c("~1", "~") else "~"
  pathCoefs <- pathCoefs[pathCoefs$op %in% operators & !is.na(pathCoefs$z),]
  pathCoefs[which(pathCoefs$op == "~1"), "rhs"] <- "(Intercept)"
  pathCoefs <- pathCoefs[order(pathCoefs$op, decreasing = TRUE),]
```